onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -552,6 +554,16 @@ def _where_input_wrangler(
 TESTED_OPS = frozenset(OPINFO_FUNCTION_MAPPING)
 
 EXPECTED_SKIPS_OR_FAILS = (
+    xfail(
+        "allclose",
+        reason="allclose return Tenosr(BOOL) but torch return Scalar(bool)",
```

Comment:
We can convert the torch output back to a tensor for comparison. I remember this has been done already?